examples/cells.ts

```diff
@@ -57,6 +58,33 @@ let started = await cells.start(<runtime.CellServiceStartRequest>{
 })
 helpers.print(started)
 
+// [ Allocate nested ]
+let nested_allocated = await cells.allocate(<runtime.CellServiceAllocateRequest>{
+    cell: runtime.Cell.fromPartial({
+        name: nestedCellName,
+        cpu: runtime.CpuController.fromPartial({
+            weight: 2, // Percentage of CPUs
+            max: 400 * (10 ** 3), // 0.4 seconds in microseconds
+        }),
+    })
+});
+helpers.print(nested_allocated)
+
+// [ Start nested ]
+let nested_started = await cells.start(<runtime.CellServiceStartRequest>{
+    cellName,
```

Comment:
Do you mean to put the exe in the nested cell?

Reply:
I didn't put a whole lot of thought into this example and I'm still not sure if we should use this example to demonstrate the list functionality. I will remove the nested exe for the sake of simplicity.